packages/plugin-essentials/sources/commands/set/version.ts

```diff
@@ -63,9 +74,17 @@ export default class SetVersionCommand extends BaseCommand {
     const report = await StreamReport.start({
       configuration,
       stdout: this.context.stdout,
+      includeLogs: !this.context.quiet,
     }, async (report: StreamReport) => {
-      report.reportInfo(MessageName.UNNAMED, `Downloading ${formatUtils.pretty(configuration, bundleUrl, `green`)}`);
-      const bundleBuffer = await httpUtils.get(bundleUrl, {configuration});
+      let bundleBuffer: Buffer;
+      if (bundleUrl.startsWith(`file://`)) {
+        report.reportInfo(MessageName.UNNAMED, `Downloading ${formatUtils.pretty(configuration, bundleUrl, FormatType.URL)}`);
+        bundleBuffer = await xfs.readFilePromise(npath.toPortablePath(bundleUrl.slice(7)));
```

Comment:
I think the slice call would be more readable if you extracted `file://` into a constant and used `.length`.